otaclient/ota_proxy/config.py

```diff
@@ -45,8 +42,15 @@ class Config:
     # DB configuration/setup
     # ota-cache table
     # NOTE: use table name to keep track of table scheme version
+    DB_THREAD_POOL_SIZE = 3
     TABLE_DEFINITION_VERSION = "v3"
-    TABLE_NAME: str = f"ota_cache_{TABLE_DEFINITION_VERSION}"
+    TABLE_NAME = f"ota_cache_{TABLE_DEFINITION_VERSION}"
+
+    # cache streaming behavior
+    STREAMING_WAIT_FOR_FIRST_BYTE = 3  # second
+    STREAMING_TIMEOUT = 10  # seconds
+    STREAMING_BACKOFF_FACTOR = 0.001  # second
+    STREAMING_WAIT_FOR_WRITER = 0  # second
```

Comment:
It is better to set non-zero not to exhaust resource, I think.

Reply:
Updated in [fad869d](https://github.com/tier4/ota-client/pull/183/commits/fad869d802f9e88bb07de73465a054a0941daf52).